tf-psa-crypto/tests/suites/test_suite_psa_crypto.data

```diff
@@ -7357,6 +7357,102 @@ PSA key agreement: FFDH RFC7919 6144 key + HKDF-SHA256: read 1+255
 
```

Comment:
Missing: tests with algorithms that don't have an interruptible implementation, and tests of the interruptible functions when interruptible functionality is disabled.

Reply:
Happy to move those tests to #9108, but are we ok moving the relevant functionality there too?